src/rustdoc-json-types/lib.rs

```diff
@@ -30,7 +30,7 @@ pub type FxHashMap<K, V> = HashMap<K, V>; // re-export for use in src/librustdoc
 /// This integer is incremented with every breaking change to the API,
 /// and is returned along with the JSON blob as [`Crate::format_version`].
 /// Consuming code should assert that this value matches the format version(s) that it supports.
-pub const FORMAT_VERSION: u32 = 42;
+pub const FORMAT_VERSION: u32 = 43;
```

Comment:
The docs for `Item::attrs` should be updated to explain what's happened here.

Reply:
Done; I also updated the `Item::attrs` comment to also explain that the `#[deprecated]` attribute is no longer present there.